src/nvme/types.h

```diff
@@ -9007,4 +9025,324 @@ struct nvme_ns_mgmt_host_sw_specified {
 };
 #endif /* SWIG */
 
+/**
+ * enum nvme_lm_select - Select (SEL): This field specifies the type of management operation to
+ * perform.
```

Comment:
Does this render the documentation correct? Is the `perform` on the short description? I recenetly learned you can add a new line in the short description, but I think then you need to have a empty line between the header and the first paragraph.

Reply:
Is does in the man page. Is there somewhere else I should look to verify?

Reply:
Check the output of `scripts/kernel-doc src/nvme/types.h`

Reply:
Looks OK to me